src/main/steps/applicant1/interim-applications/service-application-submitted/content.ts

```diff
@@ -0,0 +1,87 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import type { CommonContent } from '../../../common/common.content';
+
+const en = ({
+  serviceApplicationResponseDate,
+  serviceApplicationFeeRequired,
+  serviceApplicationDocsAllProvided,
+  serviceApplicationType,
+  referenceNumber,
+}: CommonContent) => ({
+  title: 'Application submitted',
+  introLine1: `You have submitted your application for ${serviceApplicationType}.`,
+  introLine2:
+    'Your application and help with fees reference number will be checked by court staff. You will receive an email notification confirming whether it has been accepted. Check your junk or spam email folder.',
+  sendDocumentsHeading: 'Send your evidence to the court',
+  sendDocumentsLine1: 'You now need to send us your documents. You can do this in the following ways:',
+  documentsByOnlineForm: 'Sending documents using our online form',
+  documentsByOnlineFormSteps: {
+    line1: 'You can send photographs or scans of your documents to us by',
+    line2: 'uploading them using our online form.',
+    line3:
+      'Make sure you follow the instructions on how to upload your documents carefully or they could be rejected, resulting in further delays.',
+  },
+  documentsByPost: 'Sending your documents by post',
+  documentsByPostSteps: {
+    step1: `Write your reference number on each document: ${referenceNumber}`,
+    step2: 'Post the original documents to:',
+  },
+  happensNextHeading: 'What happens next',
+  happensNextLine1: `${
+    !serviceApplicationFeeRequired && serviceApplicationDocsAllProvided
+      ? 'If your help with fees reference number is accepted, the'
+      : 'The'
+  } court will review your application and any evidence you have submitted. If your application is successful, your divorce will proceed without a response from your partner. We will then tell you when you can apply for your conditional order.`,
```

Comment:
I believe we need to conditionalize `divorce` and `partner` as we do in other places to get the correct partner and also have it work with both divorce and civil partnership?